src/main.c

```diff
@@ -409,6 +409,9 @@ static void reset_runtime_vars(data *d) {
     d->last_yaw_angle = 0;
     d->yaw_aggregate = 0;
 
+    // Reset engage_timer (keeps track of how long the board has been engaged/riding)
```

Comment:
Subjective nitpick: I find this comment to be a bit too much of a:
```c
// add two ones together
int x = 1 + 1;
```

The code is self-explanatory IMO and the comment just mostly adds unnecessary verbosity. I can certainly merge this, just that I tend to remove this kind of comments while refactoring (and just try to keep the important ones).